learning-ojs/en/statistics.md

```diff
@@ -64,6 +64,32 @@ When Geographical and Institutional Statistics have been enabled (see the [Admin
 
 This information is also available via the COUNTER SUSHI interface. See the [Administrator's Guide](https://docs.pkp.sfu.ca/admin-guide/en/statistics#counter) for details.
 
+## Institutional Statistics
+
+When making institutional statistics available to institutional users, the settings must first be altered in journal setup (requires site admin privileges). In order to do this, go to Administration -> Site Settings -> Statistics, and select the checkbox under “Institutional Subscriptions” 
+
+#### Warning
+
+> Enabling institutional statistics may have an adverse effect on the database size, and it is advised consult with your systems administrator about this change,in order to monitor how this change might affect the performance of your OJS instance going forward.  
+{:.warning}
+
+Similar to institutional subscriptions, OJS 3.4 has the ability to provide institutions with statistics regarding journal usage by institutions who are not subscribers. The data used as part of institutional statistics can be used for statistics as well as institutional subscriptions, and institutions will need to be entered into OJS first in order to capture their subscription information as well as their related statistics. 
```

Comment:
The Institutions list in 3.4 isn't parallel to the institution list for subscriptions -- they are the same lists.